src/main/java/sg/ncl/ConnectionProperties.java

```diff
@@ -160,5 +160,13 @@ public String getRegisterRequestToApplyTeam(String nclUserId) {
     public String getApproveTeam(String teamId, TeamStatus teamStatus) {
         return "http://" + sioAddress + ":" + sioPort + "/" + regEndpoint + "/teams/" + teamId + "?status=" + teamStatus;
     }
+
+    public String getExpListByTeamId(String teamId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)